src/core/utils/components/VirtualScroll.test.tsx

```diff
@@ -0,0 +1,92 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { fireEvent, render } from '@testing-library/react';
+import React from 'react';
+import VirtualScroll from './VirtualScroll';
+
+const component = (
+  <div style={{ overflow: 'auto', height: 400 }} id='scroller'>
+    <VirtualScroll
+      itemsLength={1000}
+      itemRenderer={(index) => (
+        <div key={index} className='element' style={{ height: 40 }}>{`Element${
+          index + 1
+        }`}</div>
+      )}
+    />
+  </div>
+);
+
+const renderComponent = () => {
+  return render(component);
+};
+
+const renderEmptyComponent = () => {
```

Comment:
You only use it in one place, is it really needed?

Reply:
Ok, I don't understand. Is it really empty list? Seems that it will render 1k divs

Reply:
well, I meant empty item...

Reply:
Why you need to render empty item?

Reply:
No you don't 😀 It is jsdom and you don't change the mock for client height in this test.

Reply:
oh fukin shit...................

Reply:
fixed, I think